fluentd/configs.d/openshift/filter-retag-journal.conf

```diff
@@ -60,7 +60,7 @@
   # we filter these logs through the kibana_transform.conf filter
   rewriterule1 CONTAINER_NAME ^k8s_kibana\. kubernetes.journal.container.kibana
   # mark eventrouter events
-  rewriterule2 CONTAINER_NAME ^k8s_[^_]+_logging-eventrouter-[^_]+_ kubernetes.journal.container._default_.kubernetes-event
+  rewriterule2 CONTAINER_NAME ^k8s_[^_]+_logging-eventrouter-[^_]+_ kubernetes.journal.container._default_.kubernetes.event
```

Comment:
Is there a problem with using `kubernetes-event`?

Reply:
No.  Just I noticed @jcantrill mentioned the suffix. :)